.github/workflows/decision.yml

```diff
@@ -0,0 +1,80 @@
+name: Create Branch and PR from Issue Label
+
+on:
+  issues:
+    types: [labeled]
+
+jobs:
+  create_branch_and_pr:
+    if: github.event.label.name == 'decision'
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+
+    steps:
+    - name: Authorization
+      run: |
+        # Get username of GitHub account that triggered this flow
+        opener=${{ github.actor }}
+        
+        # Get list of approved contributors
+        contributors=$(curl https://api.github.com/repos/Monstarrrr/Rebutify/contributors | jq -r '[.[] | select(.type == "User") | .login] | join(",")')
+        
+        # Convert the comma-separated string to an array
+        IFS=',' read -r -a contributor_array <<< "$contributors"
+
+        # Check if triggering user is in the array
+        if [[ ! " ${contributor_array[@]} " =~ " $opener " ]]; then
+          echo "Triggering user is unauthorized."
+          exit 1
+        fi
+    - name: Check out code
```

Comment:
What does that means?

Reply:
The user who triggered the flow by adding the decision label is unauthorized to have a branch autogenerated on their behalf.

Reply:
My comment is on line 32 😅

Reply:
Checking out the repo (aka cloning the repo).

Reply:
Oh, could you rename it to make it more clear?

Reply:
> Oh, could you rename it to make it more clear?  
>  
> Everything else look great 🚀

It's clear, git checkout.
"Check out" like checking out a book from a library. Not like checking out a cool car.

Reply:
> Actually I just thought of something, can you make sure the code allows for subfolders ``[category]/[sub_category]/.../[subject]`` ?  
> 
> Should be rare but will probably be needed at some point.

Sounds like it could be useful, can we do it in another iteration?

This needs to be tested and there are probably more fixes to do with the functionality it has currently. Don't want to bloat it too much before even getting a simple version to work.

Reply:
> > Oh, could you rename it to make it more clear?  
> >  
> > Everything else look great 🚀
> 
> It's clear, git checkout.
> "Check out" like checking out a book from a library. Not like checking out a cool car.

It wasn't clear to me because I interpreted it as an action that reviews/checks the code, rather than specifically using the ``git checkout`` command

Reply:
> > Actually I just thought of something, can you make sure the code allows for subfolders ``[category]/[sub_category]/.../[subject]`` ?  
> > 
> > Should be rare but will probably be needed at some point.
> 
> Sounds like it could be useful, can we do it in another iteration?
> 
> This needs to be tested and there are probably more fixes to do with the functionality it has currently. Bloating it too much before even getting a simple version to work will make debugging/testing dreadful.

Sure

Reply:
> > > Oh, could you rename it to make it more clear?  
> > >  
> > > Everything else look great 🚀
> > 
> > It's clear, git checkout.
> > "Check out" like checking out a book from a library. Not like checking out a cool car.
> 
> It wasn't clear to me because I interpreted it as an action that reviews/checks the code, rather than specifically using the ``git checkout`` command
> 

Calling it anything else would make it more confusing because that's the technical meaning of checkout.


Does not knowing technical term X = Technical term X is unclear?

Reply:
> Calling it anything else would make it more confusing because that's the technical meaning of checkout.
> 
> Does not knowing technical term X = Technical term X is unclear?

I was just not familiar with "check out", I am with "checkout" or "git checkout", but if that's the technical term too then sure no problem, didn't know!

Reply:
Sorry, I should have been clearer with my first answer!

Reply:
Saw your edit @Monstarrrr, done!